README.md

```diff
@@ -35,32 +50,18 @@ Codecov's Action currently supports five inputs from the user: `token`, `file`,
 
 | Input  | Description | Usage |
 | :---:     |     :---:   |    :---:   |
+| `platform` | Which platform of the uploader to use | *Required* Options include `apline` | `linux` | `macos` | `windows` |
```

Comment:
```suggestion
| `platform` | Which platform of the uploader to use | *Required* Options include `alpine` | `linux` | `macos` | `windows` |
```

Reply:
This is marked as *Required* but the code looks like it (only?) infers from `$RUNNER_OS`, and `platform` isn't listed in the inputs in `action.yml`, nor used in the tests. I'm not sure if I'm just reading it wrong, I'm not familiar with this codebase.

Reply:
@briantist, nice catch. It was `Required` in an earlier iteration of this PR, but we found a way around it via `RUNNER_OS` as you mentioned. I forgot to remove this

Reply:
Ah! Thanks @thomasrockhu , I was just thinking about this, I don't think `RUNNER_OS` would ever return `alpine`, though it would be possible to run the entire job in an alpine container wouldn't it? I wonder it makes sense to keep the input but have it to default to the (very helpful) `RUNNER_OS` value, so that `alpine` could be selected.

Again there may be a lack of understanding on my part, but might be worth testing.

Reply:
@briantist, great idea, I'll work through that this weekend before deploying next week.

Reply:
I'm actually not sure if we should allow it just yet as `node` is not on the base `alpine` image.

Reply:
@briantist ^

Reply:
@thomasrockhu even the codecov uploader documentation mentions additional dependencies for alpine so I don't think the base image would be expected to run it automatically. But in any case, up to you all, was just a suggestion. I don't currently run any jobs fully containerized to where this action would run in the container so I'm not personally affected, was just something I was thinking of.  Have a great weekend!